be/src/exprs/vectorized/array_expr.cpp

```diff
@@ -21,6 +21,9 @@ class ArrayExpr final : public Expr {
         const size_t num_elements = _children.size();
 
         size_t num_rows = 1;
+        if (num_elements == 0 && chunk) {
```

Comment:
Don‘t get this logic.

Reply:
SQL like "select **array_overlap([],c9)** from array_data_type", [] can't generate more rows than 1, It can't match with c9's size. if num_elements == 0 we should give right num_rows.

Reply:
Add comments for it

Reply:
> Add comments for it

done